test/Serializer.js

```diff
@@ -0,0 +1,118 @@
+const chaiAsPromised = require("chai-as-promised");
+const chai = require("chai");
+chai.use(chaiAsPromised);
+const {expect} = chai;
+const dynamoose = require("../dist");
+const {model} = dynamoose;
+const {Serializer} = require("../dist/Serializer");
+const utils = require("../dist/utils");
+
+describe("Serializer", () => {
```

Comment:
This needs to be improved a lot. I think we need to move a lot of this into the `Model` and `Document` tests.

Reply:
Also there are certain things not being tested. We need to increase code coverage back to 100%

Reply:
Maybe need to figure out how to test private properties in classes

Reply:
Fixed in 5102af5a74e7b98c671fdfd423ded589faea6a57